Cargo.toml

```diff
@@ -51,21 +51,21 @@ hex = { version = "0.4", default-features = false, features = ["alloc"] }
 postcard = { version = "1.0" }
 rand = { version = "0.9" }
 reqwest = "0.12"
-risc0-aggregation = { version = "0.3.2" }
-risc0-binfmt = "~1.2"
-risc0-build = { version = "~1.2", features = ["docker"] }
-risc0-build-ethereum = { version = "~1.4" }
-risc0-circuit-recursion = { version = "~1.2" }
-risc0-ethereum-contracts = { version = "~1.4" }
-risc0-zkvm = { version = "~1.2", default-features = false }
-risc0-zkp = { version = "~1.2" }
+risc0-aggregation = { version = "0.4.0-rc.2" }
+risc0-binfmt = "2.0.0"
+risc0-build = { version = "2.0.1", features = ["docker", "unstable"] }
```

Comment:
Why do we need unstable?

Reply:
our assessor guest uses the bigint accelerator that requires some experimental features to be enabled and that's why we need the unstable feature there